<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="toNFA_table">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>731</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="toDFA_table">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>731</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="miniDFA_table">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>731</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>731</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="toNFA_Button">
       <property name="text">
        <string>生成NFA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="toDFA_Button">
       <property name="text">
        <string>生成DFA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="miniDFA_button">
       <property name="text">
        <string>最小化DFA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="toText_Button">
       <property name="text">
        <string>生成程序</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>正则表达式：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>611</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>731</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="file_Button">
       <property name="text">
        <string>从文件读取</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sure_Button">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>731</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
